uni/lib/view/theme.dart

```diff
@@ -88,36 +59,12 @@ ThemeData applicationDarkTheme = ThemeData(
     tertiary: _lightGrey,
     onTertiary: _darkishBlack,
   ),
-  brightness: Brightness.dark,
-  textSelectionTheme: const TextSelectionThemeData(
-    selectionHandleColor: Colors.transparent,
-  ),
   primaryColor: _lightGrey,
-  canvasColor: _darkBlack,
-  scaffoldBackgroundColor: _darkBlack,
   cardColor: _mildBlack,
   hintColor: _darkishBlack,
   dividerColor: _strongGrey,
   indicatorColor: _lightGrey,
   primaryTextTheme: Typography().white,
   iconTheme: const IconThemeData(color: _lightGrey),
   textTheme: _textTheme.apply(bodyColor: _lightGrey),
```

Comment:
What happens if we remove these props? You can change the code throughout the app if needed.